content/guides/hosting/hosting-options/self-managed/kubernetes-operator/_index.md

```diff
@@ -465,28 +465,67 @@ global:
     secretKey: HDKYe4Q...JAp1YyjysnX
 ```
 
+The `kmsKey` must be `null`.
+
+To reference `accessKey` and `secretKey` from a secret:
+```yaml
+global:
+  bucket:
+    # Example values, replace with your own
+    provider: s3
+    name: storage.example.com
+    kmsKey: null
+    path: wandb
+    region: default
+    secret:
+      secretName: wandb-bucket
```

Comment:
suggestion: `bucket-secret` 

Just to ensure it will not clash with any secret we create by default. All of our stuff is prefixed with `wandb`